pkgs/data/fonts/victor-mono/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub }:
+
+let
+  _name = "victor-mono";
+  version = "1.2.1";
+in fetchFromGitHub {
+  name = "${_name}-${version}";
+  owner = "rubjo";
+  repo = "victor-mono";
+  rev = "v${version}";
+  postFetch = ''
```

Comment:
It would be nice if the lines were spaced out as in https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/fira/default.nix but this is just nitpicking

Reply:
Looks nicer!